src/modules/translation/languages/en.json

```diff
@@ -479,6 +479,7 @@
     "transfer": "Transfer",
     "edit": "Edit",
     "set_operator": "Set Operator",
+    "set_name": "Set name",
```

Comment:
I think this should be renamed to something else like `Set ENS` or `Set link`, because each land already has a name, that can be set by going to the `Edit` page.

Reply:
Done! thanks!